cmd/istiod/istiod.go

```diff
@@ -95,6 +95,18 @@ func main() {
 		}
 	}
 
+	// Options based on the current 'defaults' in istio.
+	// If adjustments are needed - env or mesh.config ( if of general interest ).
+
+	istiod.RunCA(istiods.GrpcServer, client, &istiod.CAOptions{
+		TrustDomain: istiods.Mesh.TrustDomain,
+		DualUse: true,
+		SelfSignedCA: true, // for existing CA - mount a secret in the expected location
+		IstioNamespace: "istio-system",
```

Comment:
Should some of these be configurable?

Reply:
Yes, at least istio-namespace will be derived from mesh.yaml, but first priority is getting things working, and avoiding making things configurable unless there is strong enough need.

Even for istio-system - I think 1.6 istiod would be fine if we only support the common case, since we'll still ship Citadel.